packages/modules/desktop-viewer-react/src/services/Initializer.ts

```diff
@@ -0,0 +1,114 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import { ClientRequestContext } from "@bentley/bentleyjs-core";
+import {
+  ElectronApp,
+  ElectronAppOpts,
+} from "@bentley/electron-manager/lib/ElectronFrontend";
```

Comment:
are these  really not exported in the main barrel?
i personally am not a big fan of imports from lib like this

Reply:
this is all that we have for these files. No barrel

Reply:
@calebmshafer heads up, we'll want to fix this core. unless there was a reason for this to begin with

Reply:
Yes, the electron-manager package is switched to handle both the frontend and backend dependencies so there isn't one single barrel file, rather two "barrels" that you need to use.